ops/docker/ci-builder/Dockerfile

```diff
@@ -1,120 +1,66 @@
-# Copy docker buildx in order to generate the absolute prestate
-# in the CI pipeline for reproducible fault proof builds
-FROM --platform=linux/amd64 docker as buildx
+###############################################################################
+#                                   BUILDX                                    #
+###############################################################################
+
+FROM --platform=linux/amd64 docker AS buildx
```

Comment:
cc @mslipper this builds but how can I actually test it in CI?

Reply:
There isn't a good way to do this - the CI builder build needs to happen in CI, then it gets tested by running it in CI. You could make a manual build, push it somewhere, update the CI config on a branch, then see if it passes.

Reply:
Ok will try this

Reply:
Ok cool, fixed the issues with CI, should be working fine now.